sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/WriteTables.java

```diff
@@ -57,8 +56,12 @@
 
 /**
  * Writes partitions to BigQuery tables.
+ *
+ * <p>The input is a list of files corresponding to a partition of a table. These files are
```

Comment:
Fixed. The problem was that the code was using a GroupByKey, when it should have been using a Reshuffle. Replacing with Reshuffle got rid of the outside Iterable<>